examples/plot_virtual_evoked.py

```diff
@@ -0,0 +1,33 @@
+"""
+=======================
+Remap MEG channel types
+=======================
+
+In this example, MEG data are remapped from one
+channel type to another. This is useful for combining statistics
+for magnetometer and gradiometers. This process can be
+computationally intensive.
+"""
+
+# Author: Mainak Jas <mainak.jas@telecom-paristech.fr>
+
+# License: BSD (3-clause)
+
+from mne.datasets import sample
+from mne import read_evokeds
+from mne.forward import compute_virtual_evoked
+
+print(__doc__)
+
+data_path = sample.data_path()
+evoked_fname = data_path + '/MEG/sample/sample_audvis-ave.fif'
+
+from_type, to_type, condition = 'grad', 'mag', 'Left Auditory'
+evoked = read_evokeds(evoked_fname, condition=condition, baseline=(-0.2, 0.0))
+
+virtual_evoked = compute_virtual_evoked(evoked, from_type=from_type,
+                                        to_type=to_type)
```

Comment:
@Eric89GXL the `report` PR will progress when I meet @dengemann . In the meanwhile, I wonder if this is to blame : https://github.com/mainakjas/mne-python/blob/master/mne/forward/_lead_dots.py#L144 for problems with mag -> grad conversion. How do I modify it for grads?